src/content/docs/en/guides/fonts.mdx

```diff
@@ -9,22 +9,22 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 
 
 
-Astro supports all common strategies for adding custom typefaces to your site design. This guide will show you two different options for including web fonts in your project.
+Astro supports all common strategies for adding custom typefaces to your site design. This guide will show you two options for including web fonts in your project.
```

Comment:
```suggestion
This guide will show you how to add web fonts to your project and use them in your components.
```
Now's a good time to revise this intro. 
- Not clear what "all common strategies" includes, and it doesn't add much. 
- "Add web fonts" matches the rest of this guide, instead of "including web fonts" 
- "two options" - also doesn't add much as we have the sidebar
- Since this is an intro to the entire page, we want to introduce more than adding, as the Tailwind section shows you an additional way to _use_ the fonts

Reply:
That's great that you spotted these aspects in the intro because they confused me as well (especially points 1 and 3) and I was going to improve it as the second step.